### PR TITLE
adds new automated tests for SPV changes

### PR DESCRIPTION
**Pull request by bhavanibandla**
### Jira link

- HOTT-3276
- HOTT-3125
- HOTT-3127

### What?

I have added/removed/altered:

- Added new automated tests for Simplified procedure values changes.

### Why?

I am doing this because:

- To ensure that new and existing functionalities work as expected.

### CODE REVIEW

File: cypress/e2e/UK/FE/spvtools-UK.cy.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  it('UK - Click and verify unit perice start date and SPV table data', function() {
+    cy.visit('tools');
+    cy.get('a[href^=\'/simplified_procedure_value\']').click();
+    cy.url().should('include', '/simplified_procedure_value');
+    cy.get('.govuk-table__head .govuk-table__header').contains('Simplified procedure code');
+    cy.get('#validity_start_date').select(0);
+    cy.contains('View rates for selected date').click();
+    cy.clickAndVerifySVPCodeCommCodeLink('2.120.1');
+  });
+  it('UK - Click and verify unit perice start date and comm code link in SPV table', function() {
```
Comment (willfish) [inline cypress/e2e/UK/FE/spvtools-UK.cy.js:29]:
```suggestion
  it('UK - Click and verify unit price start date and comm code link in SPV table', function() {
```

File: cypress/e2e/XI/FE/spvtools-XI.cy.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    cy.url().should('include', '/simplified_procedure_value');
+    cy.get('.govuk-breadcrumbs__list').contains('Simplified procedure value rates');
+    cy.contains('Northern Ireland Online Tariff');
+    cy.contains('Check simplified procedure value rates for fresh fruit and vegetables');
+    cy.contains('When to use a simplified procedure value');
+    cy.contains('Goods that are excluded');
+    cy.contains('Rates for period');
+    cy.contains('Select unit price period start date');
+  });
+  it('XI - Click and verify unit perice start date and SPV table data', function() {
```
Comment (willfish) [inline cypress/e2e/XI/FE/spvtools-XI.cy.js:20]:
```suggestion
  it('XI - Click and verify unit price start date and SPV table data', function() {
```

File: cypress/e2e/XI/FE/spvtools-XI.cy.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    cy.visit('/xi/tools');
+    cy.get('a[href^=\'/xi/simplified_procedure_value\']').click();
+    cy.url().should('include', '/simplified_procedure_value');
+    cy.get('.govuk-table__head .govuk-table__header').contains('Simplified procedure code');
+    cy.get('#validity_start_date').select(0);
+    cy.contains('View rates for selected date').click();
+    cy.clickAndVerifySVPCodeCommCodeLink('2.120.1');
+  });
+  it('XI- Click and verify unit perice start date and comm code link in SPV table', function() {
```
Comment (willfish) [inline cypress/e2e/XI/FE/spvtools-XI.cy.js]:
```suggestion
  it('XI- Click and verify unit price start date and comm code link in SPV table', function() {
```

File: cypress/e2e/XI/FE/spvtools-XI.cy.js (hunk 2 above)
Comment (bhavanibandla) [reply cypress/e2e/XI/FE/spvtools-XI.cy.js:20]: Updated and pushed changes

File: cypress/e2e/XI/FE/spvtools-XI.cy.js (hunk 3 above)
Comment (bhavanibandla) [reply cypress/e2e/XI/FE/spvtools-XI.cy.js]: Updated and pushed changes

**Review by willfish — APPROVED** [review]
LGTM

**Pull request merged**